catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/converter/type/ServiceTypeConverter.java

```diff
@@ -72,6 +72,10 @@ public ServiceTypeConverter() {
                                 .get());
                     }
 
+                    optionalBinding.get()
```

Comment:
Can we move the if block on L70-73 outside the loop and assign `optionalService.get()` to a local variable instead? Would make the loop a little shorter and save a few `if` checks.